src/structurizr/view/view.py

```diff
@@ -249,10 +273,23 @@ def find_relationship_view(
         self, source_relationship_view: RelationshipView
     ) -> Optional[RelationshipView]:
         """Find a child element view corresponding to the given relationship view."""
-        for relationship_view in self.relationship_views:
+        for relationship_view in self._relationship_views:
             if (
                 relationship_view.relationship.id
                 == source_relationship_view.relationship.id
             ):
                 return relationship_view
         return None
+
+    def check_parent_and_children_not_in_view(self, element: Element) -> None:
+        """Enusre elements can be added with parents or chldren already in the view."""
```

Comment:
```suggestion
        """Ensure that an element can be added with parents or children already in the view."""
```

Reply:
Had to tweak the wording to fit within the line length, but should be better now.